components/molecules/MainHeading/main-heading.styled.tsx

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+
+injectGlobal`@import url('https://fonts.googleapis.com/css2?family=Fredoka+One&family=Raleway&display=swap');
+
+`;
+
+export const StyledMainHeading = styled.header`
+    display: grid;
+    grid-template-columns: 3fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    gap: 0px 0px;
+    grid-template-areas:
+        'Text Emoji'
+        'job-title Emoji';
+    background-color: #ffb700;
+    margin-top: 50px;
+
+    @media screen and (max-width: 700px) {
+        grid-template-areas:
+            'Text Text'
+            'job-title job-title'
+            'Emoji Emoji';
+    }
+`;
+
+export const StyledH1 = styled.h1`
+    grid-area: Text;
+    font-family: 'Fredoka One', cursive;
+    color: White;
+    font-size: 3em;
+    margin: 0;
+    margin-left: 2rem;
+    align-self: end;
+`;
+
+export const StyledJobTitle = styled.p`
+    grid-area: job-title;
+    font-family: 'Raleway', sans-serif;
+    color: White;
+    margin: 0;
+    font-size: 1.5em;
+    margin-left: 2rem;
+    align-self: start;
+`;
+
```

Comment:
For commonly used font's it will make more sense to applies these globally so they cascade throughout the page

Reply:
Think we did this, they're not in the file anymore 👍🏼